blocks/core/ff_module/ff_module-form-preview/ff_module-form-preview.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var React = require('react');
+var InlineEdit = require('../ff_module-inline-edit/ff_module-inline-edit');
+
+module.exports = React.createClass({
+	displayName: 'FormPreview',
+	propTypes: {
+		items: React.PropTypes.array.isRequired,
+		title: React.PropTypes.string,
```

Comment:
Everything here and below is internal to each `item` in the `items` array, isn't it? So it's not a propType of this component, right?